src/IMangrove.sol

```diff
@@ -62,118 +101,334 @@ interface IMangrove is HasMgvEvents {
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@return true if locked; otherwise, false.
+  ///@dev The lock protects modifying or inspecting the offer list while an order is in progress.
   function locked(OLKey memory olKey) external view returns (bool);
 
+  ///@notice The `offerId` of the best offer on the offer list.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@return offerId The `offerId` of the best offer on the offer list.
   function best(OLKey memory olKey) external view returns (uint offerId);
 
+  ///@notice the offer list key given by the hash (if the offer list key has been activated at least once).
+  ///@param olKeyHash the hash of the offer list key.
+  ///@return olKey the olKey.
   function olKeys(bytes32 olKeyHash) external view returns (OLKey memory olKey);
 
   // # Offer view functions
 
+  ///@notice Gets an offer in packed format.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param offerId The `offerId` of the offer on the offer list.
+  ///@return offer The offer in packed format.
   function offers(OLKey memory olKey, uint offerId) external view returns (Offer offer);
 
+  ///@notice Gets an offer detail in packed format.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param offerId The `offerId` of the offer on the offer list.
+  ///@return offerDetail The offer detail in packed format.
   function offerDetails(OLKey memory olKey, uint offerId) external view returns (OfferDetail offerDetail);
 
+  ///@notice Gets both an offer and an offer detail in packed format.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param offerId The `offerId` of the offer on the offer list.
+  ///@return offer The offer in packed format.
+  ///@return offerDetail The offer detail in packed format.
   function offerData(OLKey memory olKey, uint offerId) external view returns (Offer offer, OfferDetail offerDetail);
 
   // # Governance functions
 
+  ///@notice Gets the governance address.
+  ///@return the governance address.
   function governance() external view returns (address);
 
+  ///@notice Activates an offer list.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param fee in basis points, of `olKey.outbound` given to the taker. This fee is sent to Mangrove. Fee is capped to ~2.5%.
+  ///@param density96X32 The density of the offer list used to define a minimum offer volume. See `setDensity96X32`.
+  ///@param offer_gasbase The gasbase of the offer list used to define a minimum provision necessary for offers. See `setGasbase`.
+  ///@dev The flipped offer list is expected to have the same `tickSpacing`.
```

Comment:
Not sure why this is here? To say that it's expected that the flipped offer list is also activated? (won't always be true)

Reply:
Changed to
>  ///@dev If the flipped offer list is active then the offer lists are expected to have the same `tickSpacing`.